media/css/firefox/new/scene2.scss

```diff
@@ -2,45 +2,15 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-@import '../../pebbles/includes/lib';
-@import '../../hubs/sections';
+$font-path: '/media/fonts';
```

Comment:
This diff here is the result of renaming the scss file. There are no code changes here.